aptos-move/move-examples/bonding_curve_launchpad/Move.toml

```diff
@@ -16,11 +16,11 @@ swap = "0xcafe"
 
 
 [dependencies.AptosFramework]
-git = "https://github.com/aptos-labs/aptos-core.git"
+git = "https://github.com/aptos-labs/aptos-framework.git"
 rev = "bbf569abd260d94bc30fe96da297d2aecb193644"
-subdir = "aptos-move/framework/aptos-framework"
+subdir = "aptos-framework"
 
 [dependencies.swap]
-git = "https://github.com/aptos-labs/aptos-core.git"
```

Comment:
@vgao1996 `aptos-move` subdir is not part of `"https://github.com/aptos-labs/aptos-framework.git"`, so this no longer works.